UserInterface/ViewModels/TestViewModel.cs

```diff
@@ -161,108 +167,208 @@ public string SelectedOption
             get { return _selectedOption; }
             set { _selectedOption = value; OnPropertyChanged(nameof(SelectedOption)); }
         }
+        private Ear currentAudioEar;
         #endregion
 
+        #region Commands
         private string _selectedOption = "";
         public ICommand StartTestCommand => new Command(StartTest);
         public ICommand TargetAudienceSelectedCommand => new Command(GetTest);
         public ICommand PlayFrequencyCommand { get; }
-        public ICommand SaveQuestionCommand => new Command(SaveAnswer);
-        public ICommand SaveAudioQuestionCommand => new Command(SaveAudioQuestion);
+        public ICommand SaveQuestionCommand => new Command(SaveTextAnswer);
+        public ICommand SaveAudioQuestionCommand => new Command(SaveAudioAnswer);
         public ICommand OpenTestManagementCommand => new Command(OpenTestManagement);
-
+        #endregion Commands
+        private BackgroundWorker _worker;
+        #region Constructor
         public TestViewModel(NavigationStore navigationStore)
         {
             this.navigationStore = navigationStore;
-            ShowTestExplanationView = "Visible";
+            SetTestExplanationView(VISIBLE);
             ITargetAudienceRepository targetAudienceRepository = new TargetAudienceRepository();
             targetAudienceService = new TargetAudienceService(targetAudienceRepository);
 
             ITestRepository testRepository = new TestRepository();
             testService = new TestService(testRepository);
+
+            nAudioPlayer = new BusinessLogic.Classes.NAudioPlayer();
```

Comment:
Ik zie dat je de audioplayer in de business logic hebt opgenomen, is het niet handiger om dit nog in een apart project te doen? zoals de encryption service en emailservice?